### PR TITLE
View team list and get more team info

### PR DESCRIPTION
**Pull request by Zinka010**
- Team list can be viewed under /teams
 - Click on more info to get information on the teams specific abilities (Dribbling, Passing, Shooting) etc
 - Still to do in another PR: Click on the Roster link to get list of players on that team per season
    - NOTE: The SQL and endpoint for this are written and in the PR, just the frontend remains to do

### CODE REVIEW

File: backend/src/main/java/com/example/backend/Players/PlayerController.java [hunk 1]
```diff
@@ -49,16 +52,38 @@ public ResponseEntity<String> getPlayerCount() throws ClassNotFoundException, In
...
+
+        for (int i = 0; i < IDs.size(); ++i) {
+            sb.append("player_id = ");
+            sb.append(IDs.get(i));
+            if (i != IDs.size() - 1) {
+                sb.append(" OR ");
+            }
+        }
+
+        System.out.println(sb.toString());
```
Comment (ad-world) [inline backend/src/main/java/com/example/backend/Players/PlayerController.java:69]: is this log necessary?

File: backend/src/main/java/com/example/backend/Teams/TeamController.java [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+            Connection connection = DriverManager.getConnection(Constants.url, Constants.username, Constants.password);
+            String readMessageQuery = "SELECT P.name, PTH.season FROM footyfiend.Teams as T, footyfiend.PlayerTeamHistory AS PTH,"
+                        + " footyfiend.Players AS P WHERE PTH.team_id = T.team_id AND P.player_id = PTH.player_id AND PTH.team_id = " + team_id;
+
+            // PTH.team_id = 2;
+            Statement statement = connection.createStatement();
+            ResultSet resultSet = statement.executeQuery(readMessageQuery);
+            JSONArray res = Util.resultToJsonArray(resultSet, connection);
+
+            System.out.println(res);
```
Comment (ad-world) [inline backend/src/main/java/com/example/backend/Teams/TeamController.java:93]: same with this log

File: backend/src/main/java/com/example/backend/Teams/TeamController.java [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    }
+
+    @GetMapping("/teamPlayers/{team_id}")
+    public ResponseEntity<String> getTeamPlayersById(@PathVariable String team_id) throws SQLException {
+        try {
+            Connection connection = DriverManager.getConnection(Constants.url, Constants.username, Constants.password);
+            String readMessageQuery = "SELECT P.name, PTH.season FROM footyfiend.Teams as T, footyfiend.PlayerTeamHistory AS PTH,"
+                        + " footyfiend.Players AS P WHERE PTH.team_id = T.team_id AND P.player_id = PTH.player_id AND PTH.team_id = " + team_id;
+
+            // PTH.team_id = 2;
```
Comment (ad-world) [inline backend/src/main/java/com/example/backend/Teams/TeamController.java:88]: i think this comment was here for testing, could you remove it if its not necessary?

**Review by ad-world — CHANGES_REQUESTED** [review]
left some minor nitpicky comments, other than that looks good

File: backend/src/main/java/com/example/backend/Teams/TeamController.java (hunk 3 above)
Comment (Zinka010) [reply backend/src/main/java/com/example/backend/Teams/TeamController.java]: Addressed in latest commit :)

File: backend/src/main/java/com/example/backend/Teams/TeamController.java (hunk 2 above)
Comment (Zinka010) [reply backend/src/main/java/com/example/backend/Teams/TeamController.java]: Addressed in latest commit :)

File: backend/src/main/java/com/example/backend/Players/PlayerController.java (hunk 1 above)
Comment (Zinka010) [reply backend/src/main/java/com/example/backend/Players/PlayerController.java]: Addressed in latest commit :)

**Comment by Zinka010** [comment]
> left some minor nitpicky comments, other than that looks good

Thanks for pointing these out! All addressed in https://github.com/Zinka010/footbolla/pull/30/commits/79efa0833f4a1c5c3e1bde2e5522f030fcf43d5b

**Review by ad-world — APPROVED** [review]

**Pull request merged**